docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+If the Issuing CA does not directly sign OCSP responses, it MAY make use of an OCSP Authorized Responder, as definied by [RFC 6960](https://tools.ietf.org/html/rfc6960#section-4.2.2.2). The Issuing CA of the Responder MUST be the same as the Issuing CA for the Certificates it provides responses for.
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.7.1](#71271-ocsp-responder-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.7.1 OCSP Responder Extensions
+
+| __Extension__                     | __Presence__ | __Critical__ | __Description__ |
+| ----                              | -            | -            | -----           |
+| `authorityKeyIdentifier`          | MUST         | N            | See [Section 7.1.2.9.1](#71291-authority-key-identifier) |
+| `extKeyUsage`                     | MUST         | -            | See [Section 7.1.2.7.4](#71274-extended-key-usage) |
+| `id-pkix-ocsp-nocheck`            | MUST         | N            | See [Section 7.1.2.7.5](#71275-id-pkix-ocsp-nocheck) |
+| `keyUsage`                        | MUST         | Y            | See [Section 7.1.2.7.6](#71276-key-usage) |
+| `basicConstraints`                | MAY          | Y            | See [Section 7.1.2.7.3](#71273-basic-constraints) |
+| `nameConstraints`                 | MUST NOT     | -            | - |
+| `subjectAltName`                  | MUST NOT     | -            | - |
+| `subjectKeyIdentifier`            | SHOULD       | N            | See [Section 7.1.2.9.4](#71294-subject-key-identifier) |
+| `authorityInformationAccess`      | SHOULD NOT   | N            | See [Section 7.1.2.7.2](#71272-authority-information-access) |
+| `certificatePolicies`             | -            | -            | - |
+| \ \ \ \ _Prior to 2021-10-01_     | SHOULD NOT   | N            | See [Section 7.1.2.7.7](#71277-certificate-policies) |
+| \ \ \ \ _Effective 2021-10-01_    | MUST NOT     | -            | - |
```

Comment:
Flagging so I don't forget to (obviously) update these dates as the ballot progresses ;P